index.css

```
@@ -0,0 +1,285 @@
+* {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    box-sizing: border-box;
+}
+
+html {
+    height: 100%;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+    width: 100%;
+}
+
+.mainContainer {
+    width: 400px;
+    border: 2px solid black;
+}
+
+.title-turnament {
+    background-color: gray;
+    height: 50px;
+    text-align: center;
+    border: 1px solid black;
+}
+
+.input-buttons-fields-main {
+    padding-top: 10px;
+    background-color: gray;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    align-items: center;
+    border: 1px solid black;
+    height: auto;
+}
+
+.teamA {
```

meme style a la class teamA et teamB
tu pourrais ecrire comme ca

```suggestion
.teamA, .teamB {
```

---

et du coup supprimé le paragraphe teamB